mantle/platform/api/gcloud/image.go

```diff
@@ -154,3 +154,21 @@ func (a *API) DeleteImage(name string) (*Pending, error) {
 	opReq := a.compute.GlobalOperations.Get(a.options.Project, op.Name)
 	return a.NewPending(op.Name, opReq), nil
 }
+
+func (a *API) UpdateImage(spec *ImageSpec) (*Pending, error) {
```

Comment:
Generally I prefer to have the mantle functions take the parameters and construct API specific structs themselves rather than requiring the caller to do that.

Reply:
I think I was just using  `CreateImage(spec *ImageSpec, overwrite bool)` from earlier in this file as inspiration. `CreateImage` takes information in an ImageSpec and then builds a compute.Image object out of it to make the API call with.  UpdateImage seems to be doing the same thing. What's different?

Reply:
Yeah in that respect gcloud is different from most of the other platforms (which have all been implemented at different points in time).

I'm fine punting it to a larger re-work of gcloud later to match up closer to AWS / DO that comes later.

Reply:
sounds good

Reply:
ok I changed this to take the parameters and not pass ImageSpec directly.